src/web/make-graph.rkt

```diff
@@ -276,7 +283,7 @@
                  (figcaption (p "Bits error versus " (var ,(~a var)))))]
               [else ""]))))
 
-       ,(if valid-js-prog
+       ,(if (and valid-js-prog (for/and ([p points]) (number? p)))
```

Comment:
This doesn't do anything, right? Because inputs are always flonums?

Reply:
`points` here refers to the sampled points which can be posits. Although we could let the user try out the program in floating-point, I don't think it makes much sense for posit computations, and there isn't any posit javascript library

at least not yet...